extensions/wikia/AuthModal/js/AuthModal.js

```diff
@@ -80,9 +91,32 @@ define('AuthModal', ['jquery', 'wikia.window'], function ($, window) {
 	}
 
 	return {
-		load: function (url) {
-			open();
-			loadPage(url, onPageLoaded);
-		}
+		/**
+		 * @desc launches the new auth modal if wgEnableNewAuthModal is set to true. If not, then the old UserLoginModal
+		 * is loaded.
+		 * @param {object} params:
+		 * @param {string} url - url for the page we want to load in the modal
+		 * @param {string} origin - used for tracking the source of force login modal
+		 * @param {function} successAuthCallback - callback function to be called after login
+		 */
+		load: function (params) {
+			if (typeof params.successAuthCallback !== 'function') {
```

Comment:
Thank you for that comment. I agree with it, however "21 occurrences in 12 files" :)